docs/src/test/resources/application.conf

```diff
@@ -4,3 +4,5 @@ play.modules.disabled += play.api.db.DBModule
 play.modules.disabled += play.api.db.HikariCPModule
 play.modules.disabled += com.lightbend.lagom.javadsl.persistence.jdbc.JdbcPersistenceModule
 play.modules.disabled += com.lightbend.lagom.javadsl.persistence.jpa.JpaPersistenceModule
+
+akka.test.single-expect-default = 14s
```

Comment:
Do we know that it's reading from here now?

The setting was moved in 627eff75a1f0b3eef9c2b288ec2b7375794bec9e